### PR TITLE
Updated readme for arbitrary path and url

### LINKED ISSUE #593: Update readme for new CLI cases: Download from URL and arbitrary path
We have forgotten to add this in the readme, so the users can know that they can use CLI with URL or arbitrary path.

### PR DESCRIPTION
**Pull request by davidperezgar**
Fixes #593

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ The checks can be run either using the WP Admin user interface or WP-CLI:
 * To check a plugin using WP Admin, please navigate to the _Tools > Plugin Check_ menu. You need to be able to manage plugins on your site in order to access that screen.
 * To check a plugin using WP-CLI, please use the `wp plugin check` command. For example, to check the "Hello Dolly" plugin: `wp plugin check hello.php`
     * Note that by default when using WP-CLI, only static checks can be executed. In order to also include runtime checks, a workaround is currently necessary using the `--require` argument of WP-CLI, to manually load the `cli.php` file within the plugin checker directory before WordPress is loaded. For example: `wp plugin check hello.php --require=./wp-content/plugins/plugin-check/cli.php`
+		* You could use arbitrary path or URL to check a plugin. For example, to check a plugin from a URL: `wp plugin check https://example.com/plugin.zip` or to check a plugin from a path: `wp plugin check /path/to/plugin`
```
Comment (ernilambar) [inline readme.txt]: Looks like indentation issue here. It should be aligned with `* Note that...`. May be we should add similar info in markdown file also.

File: readme.txt [hunk 2]
```diff
@@ -20,6 +20,7 @@ The checks can be run either using the WP Admin user interface or WP-CLI:
 * To check a plugin using WP Admin, please navigate to the _Tools > Plugin Check_ menu. You need to be able to manage plugins on your site in order to access that screen.
 * To check a plugin using WP-CLI, please use the `wp plugin check` command. For example, to check the "Hello Dolly" plugin: `wp plugin check hello.php`
     * Note that by default when using WP-CLI, only static checks can be executed. In order to also include runtime checks, a workaround is currently necessary using the `--require` argument of WP-CLI, to manually load the `cli.php` file within the plugin checker directory before WordPress is loaded. For example: `wp plugin check hello.php --require=./wp-content/plugins/plugin-check/cli.php`
+    * You could use arbitrary path or URL to check a plugin. For example, to check a plugin from a URL: `wp plugin check https://example.com/plugin.zip` or to check a plugin from a path: `wp plugin check /path/to/plugin`
```
Comment (ernilambar) [inline readme.txt:23]: Can you please update README.md also? It would be helpful for those who directly view GitHub reamde.

Comment (davidperezgar) [reply readme.txt:23]: ok!

**Review by ernilambar — APPROVED** [review]

**Comment by ernilambar** [comment]
@davidperezgar Can you please fix merge conflict?

**Comment by davidperezgar** [comment]
Done!

**Comment by ernilambar** [comment]
CC @swissspidy

**Review by swissspidy — APPROVED** [review]

**Pull request merged**